components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -82,7 +121,39 @@ export default function TeamSettings() {
                 title="Settings"
                 subtitle="Manage general team settings."
             >
-                <h3>Delete Team</h3>
+                <h3>Team Name</h3>
+                <p className="text-base text-gray-500 max-w-2xl">
+                    This is your team's visible name within Gitpod. For example, the name of your company.
+                </p>
+                {errorMessage && (
+                    <Alert type="error" closable={true} className="mb-2 max-w-xl rounded-md">
+                        {errorMessage}
+                    </Alert>
+                )}
+                {updated && (
+                    <Alert type="message" closable={true} className="mb-2 max-w-xl rounded-md">
+                        Profile information has been updated.
+                    </Alert>
+                )}
+                <div className="flex flex-col lg:flex-row">
+                    <div>
+                        <div className="mt-4">
+                            <h4>Name</h4>
+                            <input type="text" value={teamName} onChange={onNameChange} />
+                        </div>
+                    </div>
+                </div>
+                <div className="flex flex-row mt-8">
+                    <button
+                        className="primary"
+                        disabled={team?.name === teamName || !!errorMessage}
+                        onClick={updateTeamInformation}
+                    >
+                        Update Name
```

Comment:
**though**:: I guess this is known, but this is not changing the team slug. I'd expect to get some user feedback on this to add a section for updating the team URL. This is also out of the scope of this PR.

Reply:
I'd liek to get rid of the team URL tbh